### PR TITLE
Day 9 Edits pt. 1

### PR DESCRIPTION
**Pull request by Parzivus**
Small-ish update since I didn't want to sit on these edits for too long. Second half of day 9 will probably be finished by Thursday or Friday.

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:45]: Minor pref for with `the`

File: script/Arcueid/Day 9/09_00_ARC09_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:95]: Revert? Prev sentence ends in `.`

File: script/Arcueid/Day 9/09_00_ARC09_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:140]: There are a couple places in the text where the corpse is referred to in a very inanimate fashion, but this isn't really one of them. This sentence doesn't actually have a direct reference to the child, so we can go with they/them.

File: script/Arcueid/Day 9/09_00_ARC09_1_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1_1.txt:59]: Not sure I agree with the comma here, to me it partitions the sentence such that "even though her wounds haven't healed yet" sounds like the state of _not_ being healed would make her _not_ pay the price.

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:101]: Not sure I agree with this comma

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:140]: Mild personal pref for no `so`

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:42]: Feels like a pause too many to me

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:249]: Oops, yeah this should be "I'm not sure **what** they used"

File: script/Ciel/Day 11/11_01_CIEL11_1.txt
Comment (rschlaikjer) [inline script/Ciel/Day 11/11_01_CIEL11_1.txt:14]: Seems like an error, but we can solve that later. Assuming the rest of the edits are reexports.

File: script/Arcueid/Day 9/09_00_ARC09_1_1.txt
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_1_1.txt:59]: Right. "rambunctious + not having wounds healed" is what she's paying the price for. Not "(paying price for being rambunctious) + wounds havent healed"

File: script/Arcueid/Day 9/09_00_ARC09_2.txt
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2.txt:146]: jp is past anyways

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:16]: minor nit, but don't leave comments like this in script files. Without the context of the github pr diff, it doesn't really mean anything. Please remove before you merge

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_2B.txt:101]: I agree with it

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:9]: revert

**Review by Avuxo — APPROVED** [review]
lgtm pending comments and removal of reexports

File: script/Ciel/Day 11/11_01_CIEL11_1.txt
Comment (Parzivus) [reply script/Ciel/Day 11/11_01_CIEL11_1.txt:14]: That's correct, my folder was missing some of the Ciel files. Shouldn't show up in future edits

**Comment by Parzivus** [comment]
Response to suggestions

**Comment by Parzivus** [comment]
I'll merge this in a few hours

**Pull request merged**
